test/correctness/input_larger_than_two_gigs.cpp

```diff
@@ -11,31 +11,51 @@ void halide_error(void *ctx, const char *msg) {
 using namespace Halide;
 
 int main(int argc, char **argv) {
-    uint8_t c[4096];
-    memset(c, 42, sizeof(c));
-
-    buffer_t buf;
-    memset(&buf, 0, sizeof(buf));
-    buf.host = c;
-    buf.extent[0] = 4096;
-    buf.extent[1] = 4096;
-    buf.extent[2] = 256;
-    buf.stride[0] = 1;
-    buf.stride[1] = 0;
-    buf.stride[2] = 0;
-    buf.elem_size = 1;
-
-    Buffer param_buf(UInt(8), &buf);
+    Target t = get_target_from_environment();
+
+    Image<uint8_t> image;
     ImageParam input(UInt(8), 3);
-    input.set(param_buf);
+
+    if (t.bits == 64) {
+        image = Image<uint8_t>(4096, 4096, 256);
```

Comment:
This will also fail on the build bots. One of the 64-bit ones only has 1GB of ram (It's hard to find 64-bit arm test boards with lots of ram)